platform/sun50i/devices.c

```diff
@@ -20,10 +20,12 @@
 static struct device ccu      __device;
 static struct device msgbox   __device;
 static struct device pio      __device;
+static struct device r_axp803 __device;
 static struct device r_ccu    __device;
 static struct device r_cnt64  __device;
 static struct device r_intc   __device;
 static struct device r_pio    __device;
+static struct device r_sy8106a__device;
```

Comment:
There's no `r_` prefix for this device (the `r_` has semantic meaning–look at `platform/sun50i/include/platform/devices.h` or an SoC manual if you don't know what it is).

Reply:
done